.gitignore

```diff
@@ -1,5 +1,5 @@
 node_modules
-
+/.idea
```

Comment:
.globalignore

Reply:
See https://github.com/xFAANG/askql/issues/120 for reference

Reply:
Cool @mhagmajer ! Didn't know that :)
Solves many things in my many projects :)
So the code's working?